pkgs/servers/tailscale/default.nix

```diff
@@ -2,24 +2,24 @@
 
 buildGoModule rec {
   pname = "tailscale";
-  # Tailscale uses "git describe" as version numbers. 0.99.1-0 means
-  # "tag v0.99.1 plus 0 commits", which corresponds to rev="v0.99.1"
-  # below.
-  version = "0.99.1-0";
+  # Tailscale uses "git describe" as version numbers. 0.100.0-153
+  # means "tag v0.100.0 plus 153 commits", which corresponds to the
+  # commit hash below.
+  version = "0.100.0-153";
 
   src = fetchFromGitHub {
     owner = "tailscale";
     repo = "tailscale";
-    rev = "v0.99.1";
-    sha256 = "1kq4x5xknv0qq6n78xj5wjbf6svbdyw4nzs7z5gjb3ylj2vl97pb";
+    rev = "c06d2a851325b7b1f3f0ef99fe794df023d6ff8f";
```

Comment:
```suggestion
    rev = "v${version}";
```
besides that, LGTM

Reply:
Hm. As it happens, this works because we incorrectly created tags for a release that shouldn't have one. I think this will actually cause more problems in future, but for right now, I guess it works...